source/P0323R9.bs

```diff
@@ -905,7 +910,7 @@ template<class... Args>
     T& emplace(Args&&... args);
 ```
 
-*Constraints*: `T` is not *cv* `void` and `is_nothrow_constructible_v<T, Args...>` is `true`.
+*Constraints*: `T` is not *cv* `void` and `is_constructible_v<T, Args...>` is `true` and (`is_nothrow_constructible_v<T, Args...>` is `true` or `is_nothrow_move_constructible_v<T>` is `true` or `is_nothrow_move_constructible_v<E>` is `true`).
```

Comment:
The three ORed conditions within parentheses correspond to the three "otherwise" clauses under Effects.